src/core/integrator/euler.cpp

```diff
@@ -31,6 +31,11 @@ IntegratedActionModelEuler::IntegratedActionModelEuler(boost::shared_ptr<Differe
     set_u_lb(differential_->get_u_lb());
     set_u_ub(differential_->get_u_ub());
   }
+  if (time_step_ < 0.) {
+    time_step_ = 1e-3;
+    time_step2_ = time_step_ * time_step_;
+    throw CrocoddylException("dt has positive value, set to 1e-3");
```

Comment:
So, here the values first get initialised to a default but then it throws nonetheless. Was this meant to be a warning?

Reply:
The step time cannot be negative. If the user made stupy things, then the policy is rewrite a default dt value and let the user knows about it.

Reply:
Yes, that is clear. However: There is no point in first assigning th values and then throwing an exception. Either assign a default and show a warning, or throw an exception.

Reply:
I decided to show warning with `std::cerr`